src/components/Menu/Aria/AriaList.tsx

```diff
@@ -1,15 +1,20 @@
+import { merge } from "@utilities/merge";
 import React, { forwardRef, HTMLAttributes, ReactNode } from "react";
 
 export type AriaListProps = {
     ariaProps: HTMLAttributes<HTMLElement>;
+    noBorder?: boolean;
```

Comment:
better use the positive expression, as `border` and set it true by default

Reply:
done 👍